package.json

```diff
@@ -25,7 +25,7 @@
     "test": "ember test",
     "pretest": "npm-run-all prepublish:*",
     "posttest": "npm run clean",
-    "test:types": "ember try:each --config-path=./config/ember-try-typescript.js",
+    "test:types": "yarn --cwd type-tests && ember try:each --config-path=./config/ember-try-typescript.js",
```

Comment:
This could be slightly less clunky with the use of workspaces, but that would complicate the layout of the whole rest of the repo.

Reply:
(Maybe the `yarn` call would be better in `pretest`?)

Reply:
The slightly weird thing is that this _isn't_ necessary for `yarn test`, and `pretest` won't run if you just do `yarn test:types` locally 🤔